pcsx2/CDVD/CDVD.cpp

```diff
@@ -890,7 +890,9 @@ __fi void cdvdReadInterrupt()
 
 	if (--cdvd.nSectors <= 0)
 	{
-		cdvd.PwOff |= 1<<Irq_CommandComplete;
```

Comment:
Both methods probably work the same but cdvdSetIrq also prepares IOP for a branch test, so the flag change is potentially seen earlier then. I don't think it will make it worse..